### DISCUSSION TITLE
Ability to disable bluetooth popup menu and the show_ssid on click

### DISCUSSION THREAD

**Discussion by MagnuSiwy** [elParaguayo/qtile-extras#329] [Ideas]
Right now show_ssid makes the ssid not visible until you click the widget. Maybe the ability to disable this completely could be added? Also it's the same with the bluetooth widget's popup menu. Maybe there could be an option to disable this menu completely?

It doesn't bother me at all, just something I thought about when I first started using qtile-extras

**Comment by elParaguayo** [comment]
You can disable the click functionality by setting `mouse_callbacks` in the widget's config. Something like `mouse_callbacks={"Button1": lambda: None}`.

**Reply by MagnuSiwy** [reply]
Oh, ok, that's great, thank you!

**Comment by MagnuSiwy** [comment]
Could you add that to the widget's description in the docs for other newbies like me? I didn't think that you can just overwrite the behaviour even though it makes a lot of sense haha

**Reply by elParaguayo** [reply]
It's not specific to those widgets, you can do it to any widget that has mouse actions.
